### PR TITLE
feat(signin-screen): implement sign-in screen

### LINKED ISSUE #30: Create the LogIn screen

### PR DESCRIPTION
**Pull request by denistakeda**
Closes #30

### CODE REVIEW

**Comment by denistakeda** [comment]
It's not sexy yet but it works!

screen shot 2017-03-27 at 17 58 04

File: package.json [hunk 1]
```diff
@@ -34,12 +36,15 @@
     "nodemon": "^1.10.2",
     "null-loader": "^0.1.1",
     "pre-push": "^0.1.1",
-    "react-addons-test-utils": "^15.4.1",
+    "react": "^15.4.2",
```
Comment (sv3k) [inline package.json]: Now some deps are duplicated in both `dependencies` and `devDependencies`. Is there a reason to do that?

File: src/mobile/config/StyleConst.js [hunk 2]
```diff
@@ -1,4 +1,9 @@
+// Colors
 export const colorWhite = '#FFFFFF';
```
Comment (sv3k) [inline src/mobile/config/StyleConst.js]: Maybe rename to smth like `backgroundColor` to keep the consistency?

File: src/mobile/screens/SignInScreen/SignInScreen.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    margin: 10,
+    color: mainTextColor,
+  },
+  emailInput: {
+    height: 40,
+    borderColor: greyColor,
+    borderWidth: 1,
+    marginBottom: 10,
+  },
+  passwordInput: {
```
Comment (sv3k) [inline src/mobile/screens/SignInScreen/SignInScreen.js]: Some of the styles are exactly the same (like this one and `emailInput`). Can we just reuse them?

File: src/server/db/controllers/users.controller.js [hunk 4]
```diff
@@ -37,11 +37,11 @@ export const signUp = (_, args) => R.pipe(
...
 
-export const signIn = (_, args) => R.pipe(
-  () => User.findOne({email: args.email}),
-  then(User.comparePassword(args.password)),
-  then(generateToken)
-)(args);
+export const signIn = ({email, password}) => R.pipe(
+  () => User.findOne({email}),
+  then(user => User.comparePassword(password, user)),
+  then(user => generateToken(user))
```
Comment (sv3k) [inline src/server/db/controllers/users.controller.js]: Maybe keep `then(generateToken)`?

**Review by sv3k — CHANGES_REQUESTED** [review]

**Comment by denistakeda** [comment]
@sv3k Fixed everything. Review again please.

**Comment by sv3k** [comment]
👍

**Review by sv3k — APPROVED** [review]

**Pull request merged**